### PR TITLE
Copy puzzle info of current stack

### PR DESCRIPTION
**Pull request by Endaris**
Copies the puzzle string of the two present stacks (P1, P2) to your clipboard.

### CODE REVIEW

File: input.lua [hunk 1]
```diff
@@ -228,6 +228,19 @@ function love.keypressed(key, scancode, rep)
     love.graphics.captureScreenshot("screenshots/" .. filename)
     return
   end
+  -- copy
```
Comment (JamesVanBoxtel) [inline input.lua:231]: Now that we are getting more of these custom direct button handling cases, does it make sense to break them up into local functions?

Comment (Endaris) [reply input.lua]: done, actually realised I was missing a return in my code while breaking this up

**Review by JamesVanBoxtel — APPROVED** [review]
Tested and works great, would be nice to show onscreen the copy happened

**Comment by Endaris** [comment]
> Tested and works great, would be nice to show onscreen the copy happened

We might want to introduce some graphics snip for such button interactions in general as it's silent for screenshots as well at the moment. Going to look into that at the weekend some more.

**Comment by JamesVanBoxtel** [comment]
Sounds good, yeah it can be a later thing

**Comment by Endaris** [comment]
> Sounds good, yeah it can be a later thing

Looking at what we have for graphics and how easy it would be add a generic small fly-in animation for such notifications - cause I think it needs to be an animation to be noticed - I found myself quitting very quickly.

**Review by Shosoul — APPROVED** [review]

**Pull request merged**